pkg/secrets/generateSecret.go

```diff
@@ -5,27 +5,31 @@ import (
 	"math/big"
 )
 
-// GenerateRandomSecret generates a random secret with the specified length
-// using a mix of uppercase letters, lowercase letters, numbers, and special characters.
-func GenerateRandomSecret(length int) (string, error) {
-	const (
-		uppercase = "ABCDEFGHJKLMNPQRSTUVWXYZ"
-		lowercase = "abcdefghijkmnopqrstuvwxyz"
-		numbers   = "23456789"
-		special   = "!#$%^&*_-=+"
-	)
+const (
+	defaultLength       = 54
+	defaultCharacterSet = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789!#$%^&*_-=+"
```

Comment:
nit: do we want to remove the following characters from this list?

`IO01`

In the old code, they were not present, probably due to their similarity.

Reply:
Nice eye, copilot decided it would fill in the gaps for me I guess. I'll drop them

Reply:
On second thought, there seem to be more letters missing than that; the omissions are also not consistent between lower and upper case. I wonder if this was just an oversight initially